third_party/xla_client/mhlo_helper.cc

```diff
@@ -0,0 +1,135 @@
+#include "third_party/xla_client/mhlo_helper.h"
+#include "third_party/xla_client/xla_util.h"
+#include "tensorflow/compiler/xla/translate/hlo_to_mhlo/hlo_to_mlir_hlo.h"
+#include "tensorflow/compiler/xla/translate/mhlo_to_hlo/mlir_hlo_to_hlo.h"
+#include "tensorflow/compiler/xla/mlir_hlo/mhlo/transforms/passes.h"
+#include "mlir/IR/Verifier.h"  // from @llvm-project
+#include "mlir/Pass/PassManager.h" // from @llvm-project
+
+#include <iostream>
+
+namespace xla {
+
+void hlo_mhlo_hlo_roundtrip_helper(HloModuleProto* proto) {
```

Comment:
A function with the same name `hlo_mhlo_hlo_roundtrip_helper` seems to be defined in third_party/xla_client/computation_client.cc. I wonder where the function in this file is used.